tests/test_lib_intro.py

```diff
@@ -17,3 +17,34 @@ def test_choose_intro_emojis():
 ])
 def test_choose_intro_emoji_edge_cases(message_content, expected_emoji):
     assert expected_emoji in choose_intro_emojis(message_content)
+
+
+
+
+
+@pytest.mark.asyncio  # unfortunately ui.View() touches the event loop
+async def test_generate_intro_message_with_gh_connection_suggestion():
```

Comment:
I guess we should also have a test that the GitHub connection tutorial is not present in case there's no mention of GitHub in the text.